app/connector/sql/src/main/resources/META-INF/syndesis/connector/sql.json

```diff
@@ -151,7 +151,7 @@
                 "required": false,
                 "secret": false,
                 "tags": [],
-                "type": "string"
+                "type": "select"
```

Comment:
Is this also gonna supply the `enum` property?  The form component is gonna need some values to populate this.

Reply:
ahh - you need more then the type. What is the syntax of that?